{
  "uid" : "d3a35acd4b154e71",
  "name" : "Demographics: Multiple demographic matches shows an error that more than one result is found [Aidan-Smith-23/02/2020]",
  "fullName" : "/app/features/find_a_patient.feature:Demographics: Multiple demographic matches shows an error that more than one result is found",
  "historyId" : "1c8cb41f30fceb416baf4c1b8bdba2ab",
  "time" : {
    "start" : 1729540589333,
    "stop" : 1729540596072,
    "duration" : 6739
  },
  "status" : "failed",
  "statusMessage" : "AssertionError: assert False == True\n +  where False = check_patient_multiple_results_found_message_exists(True)",
  "statusTrace" : "fixturefunc = <function step_assert_multiple_results_found_for_patient_message at 0x7fee9c1f2e80>\nrequest = <FixtureRequest for <Function test_demographics_multiple_demographic_matches_shows_an_error_that_more_than_one_result_is_found[Aidan-Smith-23/02/2020]>>\nkwargs = {}\n\n    def call_fixture_func(\n        fixturefunc: _FixtureFunc[FixtureValue], request: FixtureRequest, kwargs\n    ) -> FixtureValue:\n        if is_generator(fixturefunc):\n            fixturefunc = cast(\n                Callable[..., Generator[FixtureValue, None, None]], fixturefunc\n            )\n            generator = fixturefunc(**kwargs)\n            try:\n                fixture_result = next(generator)\n            except StopIteration:\n                raise ValueError(f\"{request.fixturename} did not yield a value\") from None\n            finalizer = functools.partial(_teardown_yield_fixture, fixturefunc, generator)\n            request.addfinalizer(finalizer)\n        else:\n            fixturefunc = cast(Callable[..., FixtureValue], fixturefunc)\n>           fixture_result = fixturefunc(**kwargs)\n\n.tox/py311/lib/python3.11/site-packages/_pytest/fixtures.py:898: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\n    @then(\"I can see a message that more than one result was found\")\n    def step_assert_multiple_results_found_for_patient_message():\n        attach_screenshot(\"multiple_results_found_should_be_visible\")\n>       assert check_patient_multiple_results_found_message_exists(True) == True\nE       assert False == True\nE        +  where False = check_patient_multiple_results_found_message_exists(True)\n\ntests/test_find_a_patient_steps.py:207: AssertionError",
  "flaky" : false,
  "newFailed" : false,
  "newBroken" : false,
  "newPassed" : false,
  "retriesCount" : 0,
  "retriesStatusChange" : false,
  "beforeStages" : [ ],
  "testStage" : {
    "status" : "failed",
    "statusMessage" : "AssertionError: assert False == True\n +  where False = check_patient_multiple_results_found_message_exists(True)",
    "statusTrace" : "fixturefunc = <function step_assert_multiple_results_found_for_patient_message at 0x7fee9c1f2e80>\nrequest = <FixtureRequest for <Function test_demographics_multiple_demographic_matches_shows_an_error_that_more_than_one_result_is_found[Aidan-Smith-23/02/2020]>>\nkwargs = {}\n\n    def call_fixture_func(\n        fixturefunc: _FixtureFunc[FixtureValue], request: FixtureRequest, kwargs\n    ) -> FixtureValue:\n        if is_generator(fixturefunc):\n            fixturefunc = cast(\n                Callable[..., Generator[FixtureValue, None, None]], fixturefunc\n            )\n            generator = fixturefunc(**kwargs)\n            try:\n                fixture_result = next(generator)\n            except StopIteration:\n                raise ValueError(f\"{request.fixturename} did not yield a value\") from None\n            finalizer = functools.partial(_teardown_yield_fixture, fixturefunc, generator)\n            request.addfinalizer(finalizer)\n        else:\n            fixturefunc = cast(Callable[..., FixtureValue], fixturefunc)\n>           fixture_result = fixturefunc(**kwargs)\n\n.tox/py311/lib/python3.11/site-packages/_pytest/fixtures.py:898: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\n    @then(\"I can see a message that more than one result was found\")\n    def step_assert_multiple_results_found_for_patient_message():\n        attach_screenshot(\"multiple_results_found_should_be_visible\")\n>       assert check_patient_multiple_results_found_message_exists(True) == True\nE       assert False == True\nE        +  where False = check_patient_multiple_results_found_message_exists(True)\n\ntests/test_find_a_patient_steps.py:207: AssertionError",
    "steps" : [ {
      "name" : "Given I am on the find a patient by demographics page",
      "time" : {
        "start" : 1729540589333,
        "stop" : 1729540594408,
        "duration" : 5075
      },
      "status" : "passed",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ ],
      "stepsCount" : 0,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : false,
      "hasContent" : false
    }, {
      "name" : "And I enter the mandatory patient details Aidan, Smith, and 23/02/2020",
      "time" : {
        "start" : 1729540594408,
        "stop" : 1729540595669,
        "duration" : 1261
      },
      "status" : "passed",
      "steps" : [ ],
      "attachments" : [ {
        "uid" : "2d3e38893a849d41",
        "name" : "dev_chrome_130.0.6723.58_add_mandatory_patient_information.png",
        "source" : "2d3e38893a849d41.png",
        "type" : "image/png",
        "size" : 30888
      } ],
      "parameters" : [ ],
      "stepsCount" : 0,
      "attachmentsCount" : 1,
      "shouldDisplayMessage" : false,
      "hasContent" : true
    }, {
      "name" : "When I click the search button",
      "time" : {
        "start" : 1729540595669,
        "stop" : 1729540595947,
        "duration" : 278
      },
      "status" : "passed",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ ],
      "stepsCount" : 0,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : false,
      "hasContent" : false
    }, {
      "name" : "Then I can see a message that more than one result was found",
      "time" : {
        "start" : 1729540595948,
        "stop" : 1729540596072,
        "duration" : 124
      },
      "status" : "failed",
      "statusMessage" : "assert False == True\n +  where False = check_patient_multiple_results_found_message_exists(True)",
      "statusTrace" : "AssertionError: assert False == True\n +  where False = check_patient_multiple_results_found_message_exists(True)\n",
      "steps" : [ ],
      "attachments" : [ {
        "uid" : "1ca645913a3e7387",
        "name" : "dev_chrome_130.0.6723.58_multiple_results_found_should_be_visible.png",
        "source" : "1ca645913a3e7387.png",
        "type" : "image/png",
        "size" : 31283
      } ],
      "parameters" : [ ],
      "stepsCount" : 0,
      "attachmentsCount" : 1,
      "shouldDisplayMessage" : true,
      "hasContent" : true
    } ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 4,
    "attachmentsCount" : 2,
    "shouldDisplayMessage" : true,
    "hasContent" : true
  },
  "afterStages" : [ ],
  "labels" : [ {
    "name" : "host",
    "value" : "53d063788252"
  }, {
    "name" : "thread",
    "value" : "1097-MainThread"
  }, {
    "name" : "framework",
    "value" : "pytest-bdd"
  }, {
    "name" : "language",
    "value" : "cpython3"
  }, {
    "name" : "feature",
    "value" : "Find a patient"
  }, {
    "name" : "resultFormat",
    "value" : "allure2"
  } ],
  "parameters" : [ {
    "name" : "dob",
    "value" : "23/02/2020"
  }, {
    "name" : "firstName",
    "value" : "Aidan"
  }, {
    "name" : "lastName",
    "value" : "Smith"
  } ],
  "links" : [ ],
  "hidden" : false,
  "retry" : false,
  "extra" : {
    "severity" : "normal",
    "retries" : [ ],
    "categories" : [ {
      "name" : "Product defects",
      "matchedStatuses" : [ ],
      "flaky" : false
    } ],
    "tags" : [ ]
  },
  "source" : "d3a35acd4b154e71.json",
  "parameterValues" : [ "23/02/2020", "Aidan", "Smith" ]
}